packages/frontend/src/components/NavBar/SettingsMenu/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Menu, PopoverPosition } from '@blueprintjs/core';
+import { MenuItem2, Popover2 } from '@blueprintjs/popover2';
+import { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useActiveProjectUuid } from '../../../hooks/useProject';
+import { useApp } from '../../../providers/AppProvider';
+
+const SettingsMenu: FC = () => {
+    const { user } = useApp();
+    const history = useHistory();
+    const activeProjectUuid = useActiveProjectUuid();
+
+    const userCanManageProjects = user.data?.ability?.can('manage', 'Project');
```

Comment:
Should we check if they can manage the active project? eg:
```
const canUpdateTableConfiguration = ability.can(
        'update',
        subject('Project', {
            organizationUuid: user.data?.organizationUuid,
            projectUuid,
        }),
    );
```

Reply:
@ZeRego 💯